### PR TITLE
adjusting note and issues

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
First attempt to implement the decisions taken in [DCAT call 11-11-2020](https://www.w3.org/2020/11/11-dxwgdcat-minutes.html).
Please double-check if anything is missing.

### CODE REVIEW

File: dcat/index.html [hunk 1]
```diff
@@ -3504,7 +3474,7 @@ <h2>How to specify dataset series</h2>
 <h2>Property values inheritance in dataset series</h2>
 
 <aside class="ednote">
-<p>The approach proposed in this section is being reviewed.</p>
+<p>The approach proposed in this section is being reviewed.  </p>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]: @andrea-perego , @agbeltran : I wonder if we want to keep this editorial note?

Comment (andrea-perego) [reply dcat/index.html]: I would drop it, and include instead the reference to https://github.com/w3c/dxwg/issues/1273

File: dcat/index.html [hunk 2]
```diff
@@ -3531,6 +3501,7 @@ <h2>Property values inheritance in dataset series</h2>
 
 <aside class="ednote">
```
Comment (andrea-perego) [inline dcat/index.html]: I would remove the EDNOTE, and keep only the issue.

**Review by andrea-perego — CHANGES_REQUESTED** [review]
I added some inline comments.

File: dcat/index.html (hunk 2 above)
Comment (riccardoAlbertoni) [reply dcat/index.html]: Thanks, Andrea. I think I have included your suggestion.

Comment (andrea-perego) [reply dcat/index.html]: Thanks

**Comment by andrea-perego** [comment]
@riccardoAlbertoni , I created a PR with some additional changes - https://github.com/w3c/dxwg/pull/1279

File: dcat/index.html [hunk 3]
```diff
@@ -3279,29 +3282,6 @@ <h2>Versioning</h2>
...
-<!--
-<aside class="note">
-<p>
-In this section it is planned to describe some patterns for describing Dataset and/or Distribution versions.
-See the wiki page on <a href="https://github.com/w3c/dxwg/wiki/Dataset-versioning">Dataset versioning</a> for more discussion.
-</p>
-</aside>
--->
-
-<aside class="ednote">
```
Comment (andrea-perego) [inline dcat/index.html:3299]: I would keep this EDNOTE, removing the reference to the issue.

**Comment by agbeltran** [comment]
Adding link to live version to facilitate the review: https://rawgit.com/w3c/dxwg/dcat-adjustingReferenceToIssues/dcat/index.html

File: dcat/index.html [hunk 4]
```diff
@@ -12,6 +12,9 @@
 <body>
 
 <!-- Disclaimer -->
+    <aside class="ednote">
+    <p>This document presents advancements in the discussion made on DCAT 3 within DXWG group. It complements the DCAT 2 [[?VOCAB-DCAT-2-20200204]] with guidelines on Versioning and Dataset series (see sections <a href="#dataset-versions"></a> and <a href="#dataset-series"></a>). </p>
```
Comment (agbeltran) [inline dcat/index.html]: I suggest changing 'complements' to 'extends'

**Review by agbeltran — CHANGES_REQUESTED** [review]

File: dcat/index.html (hunk 3 above)
Comment (riccardoAlbertoni) [reply dcat/index.html:3299]: Done

File: dcat/index.html (hunk 4 above)
Comment (riccardoAlbertoni) [reply dcat/index.html]: Done

**Comment by riccardoAlbertoni** [comment]
@andrea-perego  and @agbeltran:
I have included your suggestions. If no other adjustments are planned I guess we can approve this PR and merge it

**Review by agbeltran — APPROVED** [review]

**Review by andrea-perego — APPROVED** [review]

**Pull request merged**